sdk_container/src/third_party/coreos-overlay/sys-apps/baselayout/baselayout-9999.ebuild

```diff
@@ -1,138 +1,69 @@
 # Copyright 1999-2013 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 EGIT_REPO_URI="https://github.com/flatcar/baselayout.git"
 
 if [[ "${PV}" == 9999 ]]; then
 	inherit git-r3
-	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+	KEYWORDS="~amd64 ~arm64"
 else
-	EGIT_COMMIT="937a45faef0f7fa88d3d2c3f7ba60a7f3e2e82f7" # flatcar-master
+	EGIT_COMMIT="8e203be908ca99dee55d03ac4cf2dfe965eac065" # krnowak/merged-usr
 	SRC_URI="https://github.com/flatcar/baselayout/archive/${EGIT_COMMIT}.tar.gz -> flatcar-${PN}-${EGIT_COMMIT}.tar.gz"
 	S="${WORKDIR}/${PN}-${EGIT_COMMIT}"
-	KEYWORDS="amd64 arm arm64 x86"
+	KEYWORDS="amd64 arm64"
 fi
 
-TMPFILES_OPTIONAL=1
-inherit multilib systemd tmpfiles
+inherit multilib
 
-DESCRIPTION="Filesystem baselayout for CoreOS"
-HOMEPAGE="http://www.coreos.com/"
+DESCRIPTION="Filesystem baselayout for Flatcar"
+HOMEPAGE="https://www.flatcar.org/"
 
 LICENSE="GPL-2"
 SLOT="0"
-IUSE="cros_host symlink-usr"
+IUSE="cros_host"
 
 # This version of baselayout replaces coreos-base
-DEPEND="sys-apps/systemd
+DEPEND="
+	sys-apps/systemd
```

Comment:
This systemd dependency can cause issues, so I'd appreciate it if you could change it to `RDEPEND` only.

Reply:
What about dropping it? The `net-dns/libidn2` dependency also looks weird here. The rest of deps are conflicts to some old versions, which could be dropped too, so maybe I'll delete the DEPEND variable entirely?

Reply:
Technically, systemd is needed so that systemd-tmpfiles can process these files at runtime. You could argue that we only ever use this package with systemd-based profiles and therefore systemd is part of the system set.

I don't know what libidn2 would have been used for.

Reply:
> This systemd dependency can cause issues, so I'd appreciate it if you could change it to RDEPEND only.

What issues?

Reply:
It should have been `BDEPEND` rather than `DEPEND`. IIRC, it caused cyclic dependency issues when I cross-compiled stage3 for the arm64 SDK.